packages/provisioning/src/cli/Provisioning.definition.ts

```diff
@@ -20,6 +20,7 @@ const definition: ICommandDefinition = {
     name: "provisioning",
     aliases: ["pv"],
     type: "group",
+    summary: "Perform z/OSMF provisioning tasks",
```

Comment:
We need to implement these changes in LTS incremental.
@ChrisBoehmCA, any thoughts?

Reply:
I can open a new PR for LTS incremental.

Reply:
Thank you :)